components/Form.js

```diff
@@ -26,7 +26,7 @@ export default function Form() {
   const picID = crypto.randomBytes(16).toString("hex");
 
   function alltimeFavorites() {
```

Comment:
You don't need this function.

In line 100 there is a StyledButton calling this function and redirecting to another page.
This should be done with the an anchor tag (or with the Link component from Next.js).

You only want to redirect the user programmatically, if some other lines of code need to be executed beforehand. For instance when you submit a form, you want the to do something with the submitted data before going to a different page.

An anchor tag is semantically used for links. And the Link component provides us with some other cool features like preloading the page for a better user experience

Reply:
but i wouldnt wrap a button in an anchor tag?!

Reply:
correct! It should not be a button but only a Link.

`<Link href="/AlltimeFavorites">alltimeFavorites</Link>`